doc/source/user/absolute-beginners-outline.rst

```diff
@@ -0,0 +1,188 @@
+****************************************
+NumPy: The Absolute Basics for Beginners
+****************************************
+
+This is a working outline for a future section introducing NumPy to absolute beginners. If you have comments or suggestions, please don’t hesitate to reach out!
+
+
+
+- How to install NumPy
+  
+  - various operating systems 
+
+  - If you don't have Python yet, consider using Anaconda as the easiest way to get started
+
+    - The good thing about getting this Python distribution is is the fact that you don’t need to worry too much about separately installing NumPy or any of the major packages that you’ll be using for your data analyses, such as pandas, Scikit-Learn, etc.
+    
+    - If you do have Python, you can install NumPy with `conda install numpy` or `pip install numpy`
+    
+    - For more details, see the `Installation` section
+
+- How to import NumPy
+
+::
+
+  import numpy as np 
+
+(We shorten "numpy" to "np" in order to save time and also so that code is standardized so that anyone working with your code can easily understand and run it.)
+
+- What is an array?
+
+  - An array is a central data structure of the NumPy library. It's a grid of values and it contains information about the raw data, how to located an element, and how to interpret an element. All of the values in an array should be the same type and an array is indexed by a tuple of nonnegative integers. The *rank* of the array is the number of dimensions. The *shape* of the array is a tuple of integers giving the size of the array along each dimension.
+
```

Comment:
"It's a grid of values and it contains information about the raw data, how to located an element, and how to interpret an element. All of the values in an array should be the same type and an array is indexed by a tuple of nonnegative integers. " 

The statement about "indexed by a tuple of non-negative integers" is only one way to index, it can be indexed by booleans, by another array, or by integers.

suggested change: 
```
"It's has two main components: a grid of elements that can be indexed in `various ways <indexing link>`. The elements are all of the same type, referred to as the array `dtype`.
```

Reply:
Ahh, I see you took the indexing text from the [quickstart page](https://numpy.org/devdocs/user/quickstart.html), in my opinion it is inaccurate there as well.

Reply:
Thank you so much! I'll get this updated!